data/json/items/tool/cooking.json

```diff
@@ -52,6 +52,21 @@
     "color": "light_cyan",
     "flags": [ "ALLOWS_REMOTE_USE" ]
   },
+  {
+    "id": "screw_press",
+    "type": "TOOL",
+    "name": { "str": "screw press", "str_pl": "screw presses" },
+    "description": "This is a crafting_pseudo_item if you have it something is wrong.",
```

Comment:
but that's what you get from deconstructing the screw press furniture

Reply:
thanks for taking the time to look at this, appreciate it. I'll try to make the adjustments in the coming days.

Reply:
I changed what you get from deconstruct to the right item now, see below.